package.json

```diff
@@ -26,7 +26,7 @@
   },
   "homepage": "https://github.com/scality/s3utils#readme",
   "dependencies": {
-    "arsenal": "github:scality/arsenal#development/8.1",
+    "arsenal": "github:scality/arsenal#07a7d5e",
```

Comment:
Curious as to the decision to use Github urls and git hashes instead of semantic versioning like everything else?

Reply:
I don't think we maintain the semantic versions regularly on each package, hence we use the commit hashes instead. It might not be the best I agree, here I changed it because other projects rely on commit hashes not dev branch names so it's more consistent and works even if the dev branch hasn't been updated yet.

Maybe @rahulreddy can give a more elaborate answer if you're interested, as this topic has some history.

Reply:
There's also the general unreliability of yarn (which is an improvement over npm, but still sometimes generates installations that are not identical to previous ones, everything else being equal)

Reply:
Hmm I know years ago this was difficult for npm / yarn to properly tree out all dependencies but as far as I know now this should generally be a non-issue. I would generally recommend using semver for 99% of use cases at this point. Calver and hash versioning have use cases but not for installed packages IMO. 

This all probably relates to release management that I have no clue about yet so obviously take anything I say with "sure but you are missing this and that".

Reply:
It was not a non-issue for us, and not years ago.

Reply:
OK, no worries. I had assumed there was some issue but that I just didn't have enough information. Curious as to what may be causing these as I have not run into it myself before.